backend-project/small_eod/files/models.py

```diff
@@ -10,10 +10,10 @@ class File(models.Model):
     name = models.CharField(
         max_length=100, verbose_name=_("Name"), help_text=_("Name of file.")
     )
-    letter = models.ForeignKey(
+    letters = models.ForeignKey(
```

Comment:
Dlaczego zmieniasz to na liczbe mnogą, skoro to jest jedynie ForeignKey?

Reply:
Przy zmianach nazw pól nie kierowałem się tylko zależnościami, a również poprawnością semantyczną. To też miało być jednym z założeń zmian.

Reply:
Na czym polega poprawność sematyczna w tym przypadku?